### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by marceloyuela**
Good catch Jim, that would have been a big mistake.
Notes from the (wrong) pull request:

The primary meat of this pull request is added Verbose capability for most cmdlets. The "get" cmdlets don't have it because they are much quicker and don't really have any steps to them.

All cmdlets that take time now display some progress.

The "Fixed Import Tests" commit happened because the instances lost SUPER Privilege for some reason (update with the API?), so the files being imported had to be changed slightly.

### CODE REVIEW

**Comment by marceloyuela** [comment]
Sorry for that other pull request- glad you caught the mistake @ILMTitan

File: Google.PowerShell/Sql/GcSqlInstanceCmdlets.cs [hunk 1]
```diff
@@ -153,6 +152,7 @@ public class AddGcSqlInstanceCmdlet : GcSqlCmdlet
         protected override void ProcessRecord()
         {
             InstancesResource.InsertRequest request = Service.Instances.Insert(InstanceConfig, Project);
+            WriteVerbose(string.Format("Adding instance '{0}' to Project '{1}'.", InstanceConfig.Name, Project));
```
Comment (ILMTitan) [inline Google.PowerShell/Sql/GcSqlInstanceCmdlets.cs]: I think we prefer the less verbose $"Adding instance '{InstanceConfig.Name}' to Project '{Project}'."

File: Google.PowerShell/Sql/GcSqlCmdlet.cs [hunk 2]
```diff
@@ -63,13 +63,17 @@ public Operation WaitForSqlOperation(Operation op)
                         break;
                     }
             }
+            WriteProgress(new System.Management.Automation.ProgressRecord(0, string.Format("Waiting for operation '{0}' to finish.", op.OperationType), "Waiting"));
```
Comment (ILMTitan) [inline Google.PowerShell/Sql/GcSqlCmdlet.cs]: Have you considered moving the progress bar rather than changing the text?

**Comment by ILMTitan** [comment]
I see a lot of string.Format that could probably simplified. Also, perhaps we should move the progress bar rather than changing the text. It would be similar, but instead of %=4 we would have a %=100.

File: Google.PowerShell/Sql/GcSqlCmdlet.cs (hunk 2 above)
Comment (marceloyuela) [inline Google.PowerShell/Sql/GcSqlCmdlet.cs]: Will look into this.

**Comment by ILMTitan** [comment]
I am concerned the progress bar will not disappear without writing a ProgressRecord of RecordType = Complete. If you are sure it does, LGTM.

**Comment by marceloyuela** [comment]
Updated that just in case. It seemed to complete regardless but I'd rather not risk sing up extra resources.

**Comment by ILMTitan** [comment]
LGTM

**Pull request merged**